### PR TITLE
blog(embertimes156): econn with typescript blog

### PR DESCRIPTION
**Pull request by chrisrng**

### CODE REVIEW

File: source/2020-07-17-the-ember-times-issue-156.md [hunk 1]
```diff
@@ -91,14 +91,30 @@ READMORE
...
+
+export default Component.extend({
+  myTask: task(function*() {
+    yield foo;
+    return bar;
+  })
+}
+```
+
+The blog post goes through how using Ember Octane and TypeScript we need to use Native Class Syntax and so the [ember-concurrency-decorators](https://github.com/machty/ember-concurrency-decorators) package was created to make this work for ember-concurrency.
```
Comment (abhilashlr) [inline source/2020-07-17-the-ember-times-issue-156.md]:
```suggestion
The blog post goes through how using Ember Octane and TypeScript, we need to use Native Class Syntax. The [ember-concurrency-decorators](https://github.com/machty/ember-concurrency-decorators) package was created to make this work for ember-concurrency.
```

**Review by abhilashlr — COMMENTED** [review]
Thanks for the amazing write up @chrisrng 🎉 . Added a minor comment, feel free to discard if it isn't reading great 👍

File: source/2020-07-17-the-ember-times-issue-156.md (hunk 1 above)
Comment (chrisrng) [reply source/2020-07-17-the-ember-times-issue-156.md]: Thanks! accepting this suggestion

File: source/2020-07-17-the-ember-times-issue-156.md [hunk 2]
```diff
@@ -91,14 +91,30 @@ READMORE
...
+
+export default Component.extend({
+  myTask: task(function*() {
+    yield foo;
+    return bar;
+  })
+}
+```
+
+The blog post goes through how using Ember Octane and TypeScript, we need to use Native Class Syntax. The [ember-concurrency-decorators](https://github.com/machty/ember-concurrency-decorators) package was created to make this work for ember-concurrency.
```
Comment (ijlee2) [inline source/2020-07-17-the-ember-times-issue-156.md]:
```suggestion
The blog post goes through how to use ember-concurrency with Ember Octane and TypeScript. For Octane, we need to use Native Class Syntax. The [ember-concurrency-decorators](https://github.com/machty/ember-concurrency-decorators) package was created to make this work for ember-concurrency.
```

File: source/2020-07-17-the-ember-times-issue-156.md [hunk 3]
```diff
@@ -91,14 +91,30 @@ READMORE
...
+
+However, decorators cannot change the type of the thing they decorate. So the [ember-concurrency-ts](https://github.com/chancancode/ember-concurrency-ts) package was created which provides a couple of utility functions such as `taskFor` for using TypeScript with ember-concurrency.
```
Comment (ijlee2) [inline source/2020-07-17-the-ember-times-issue-156.md]:
```suggestion
However, decorators cannot change the type of the thing they decorate. To allow TypeScript, the [ember-concurrency-ts](https://github.com/chancancode/ember-concurrency-ts) package was created to provide a couple of utility functions such as `taskFor`. 
```

**Review by ijlee2 — APPROVED** [review]
Hi, Chris. The write-up looks great. I made a couple of additional suggestions.

Before you merge this PR, can you rebase your branch so that you have the latest commits from the `master` branch? The 2 latest commits by Amy should fix the Travis build failure (Alex lint errors) that we're seeing in your PR.

Thanks for covering the article! 🧡

**Pull request merged**